dlt/destinations/job_client_impl.py

```diff
@@ -42,8 +42,13 @@ def __init__(self, file_path: str, sql_client: SqlClientBase[Any]) -> None:
         # execute immediately if client present
         with FileStorage.open_zipsafe_ro(file_path, "r", encoding="utf-8") as f:
             sql = f.read()
+        if sql_client.capabilities.supports_multiple_statements:
+            statements = [sql]
+        else:
+            statements = sql.splitlines()
```

Comment:
I changed the `SqlMergeJob` implementation to write one statement per line in the file.   
Found it kinda sketchy to split on `;` since it can appear anywhere in strings and comments too. Maybe overkill to consider that here...  

But will remove this anyway due to https://github.com/dlt-hub/dlt/pull/414#discussion_r1233439895